home.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My component library</title>
+    <script src="https://kit.fontawesome.com/917a992f26.js" crossorigin="anonymous"></script>
```

Comment:
meta can be used to give more info on the site

Reply:
not using component library for styles?

Reply:
I am using component library. I am just copy pasting styles from it. Component library is also in PR.